code/processes/segmentedtickerplant.q

```diff
@@ -83,5 +83,38 @@ init:{
   .stplg.init[string .proc.procname];
  };
 
-// Have the init function called from torq.q
+// Loads the striping.json config file and logs any problems
+configload:{[scpath]
+     @[jsonchecks; 
+       .stpps.stripeconfig:@[{.j.k read1 x};scpath;{.lg.o[`configload;"Failed to load in striping.json file: ",x]}];
+       {.lg.o[`configload;"Datastriping is not enabled due to problems with striping.json"]}]
```

Comment:
probably worth including the error?

```suggestion
       {.lg.o[`configload;"Datastriping is not enabled due to problems with striping.json: ",x]}]
```

Reply:
I believe the error is logged in an earlier log message. This log message is just to log that datastriping will not be on as a consequence so the user knows normal TorQ is running instead of TorQv5 functionality. (I believe Jonny said the requirements were to have normal TorQ runing if any errors occur)

Reply:
I think there's probably no harm including it here as well?

Reply:
I will just put in one log message instead.